Frontend/src/pages/create.js

```diff
@@ -114,3 +121,24 @@ export default function Create() {
     </div>
   );
 }
+
+
+export async function getServerSideProps(context) {
+  const { req } = context;
+  const token = req.cookies["token"]; // Replace "your_cookie_name" with your actual cookie name
+
+  if (!await isAuthenticated(token)) {
+    // If the user is not authenticated, redirect them to the login page
+    return {
+      redirect: {
+        destination: "/login",
+        permanent: false,
+      },
+    };
+  }
+
+  // If the user is authenticated, render the Portfolio page
```

Comment:
Is this an old comment? Confused about the "render Portfolio page" part

Reply:
its due to me copy and pasting from the portfolio page... should say "Create page"